lib/src/qr_code_scanner.dart

```diff
@@ -325,11 +327,11 @@ class QRViewController {
 
   /// Updates the view dimensions for iOS.
   static Future<void> updateDimensions(GlobalKey key, MethodChannel channel,
-      {QrScannerOverlayShape overlay}) async {
+      {QrScannerOverlayShape? overlay}) async {
     if (defaultTargetPlatform == TargetPlatform.iOS) {
-      // Add small delay to ensure the renderbox is loaded
+      // Add small delay to ensure the render box is loaded
       await Future.delayed(Duration(milliseconds: 100));
-      final RenderBox renderBox = key.currentContext.findRenderObject();
+      final renderBox = key.currentContext!.findRenderObject() as RenderBox;
```

Comment:
Damn, this code seem dangerous 😨
Maybe `updateDimensions` should return `Future<bool>` with `false` value if `renderbox` or `key.currentContext` are not available?

Reply:
When trying the latest null-safe build, I hit this:
```
[VERBOSE-2:ui_dart_state.cc(186)] Unhandled Exception: Null check operator used on a null value
#0      QRViewController.updateDimensions (package:qr_code_scanner/src/qr_code_scanner.dart:320:44)
```

Reply:
Updated to Future<bool> and added null check